README.md

```diff
@@ -1,25 +1,218 @@
+<div align="center">
+<h1> donkeycar LPH </h1>
 
-## Documentation :
+![repo-size](https://img.shields.io/github/repo-size/mdl29/donkeycarLPH?style=flat-square)
+![contributor](https://img.shields.io/github/contributors/mdl29/donkeycarLPH?style=flat-square)
+![commitActivity](https://img.shields.io/github/commit-activity/m/mdl29/donkeycarLPH?logo=github&style=flat-square)
+![mainLanguage](https://img.shields.io/github/languages/top/mdl29/donkeycarLPH?color=yellow&style=flat-square)
 
-- [Framapad](https://annuel.framapad.org/p/LPH-donkeycar)
+### In this project we present a solution to industrialize the production of donkeycar car raspberry pi images and allow everyone to be able to install easily the donkeycar environnement. We have also added some tools like our manager to make public demonstrations, led several donkeycars to the public having now knowledge of the project, register for a driving session (with an estimated waiting time), drive with multiple cars at the same time and see in live on a huge screen their time lapses, start a recorded driving then view their own model in action (without handling transfers ...).
+</div>
+<br>
 
-## Software :
+ > 💡 Inspirated and use the original [donkeycar project](https://github.com/autorope/donkeycar)
+ 
+ <br>
 
-titre | description | fichier
- ---: | :---: | :--- 
-Doc pour le boutton poussoir | Créer un boutton permettant d'éteindre une voiture | [documentation](doc/boutton/Boutton_poussoir.md)
-Doc pour les backups | Comment créer sa backup pour sa raspberry pi | [documentation](doc/backup/car-backup-procedure.md)
-Doc pour Ansible | Comment installer donkeycar sur des voitures ( plus nos features ) avec Ansible en ssh | [documentation](ansible/README.md)
-Doc pour packer | Comment generer une image propre pour les voitures avec packer | [documentation](packer/README.md)
+<details>	
+  <summary><b> 🚀 How to use our project</b></summary>
 
-## Hardware :
+## 🧰 Build your own donkeycar 
 
-titre | description | fichier
- ---: | :---: | :--- 
-Shéma des branchements| Schema de la connectique de la donkeycar | [image](doc/schema/schema-electronique.png) / [Fichier Fritzing](doc/schema/shema-electronique.fzz)
-Antivol | Antivol pour manette PS4 | [doc](3dDesigns/anti-theft-ps4/README.md)
-Chassis pour voiture V4 | Picots pour adaptation de la voiture au chassis donkeycar | [doc](3dDesigns/car-chassis/README.md)
-Camera cap holder | A camera cap holder on the chassis for avoid to lose ours camera caps | [doc](3dDesigns/camera-cover/README.md)
+***
 
+Before using our project you need to build your donkeycar.
+If you don't have a donkeycar, please follow this [tutorial](https://docs.donkeycar.com/guide/build_hardware/) made by donkeycar community
+
+### **Tips :**
+If you using wide angle camera like [this](https://www.kubii.fr/cameras-capteurs/2207-module-camera-5mp-grand-angle-kubii-3272496011205.html), we have modelize on the chassis a camera cap holder. Convenient to avoid losing it
```

Comment:
if you are using a wide angle camera [...] we have modeled a camera cap holder on the chassis